.circleci/build_dependencies.sh

```diff
@@ -15,39 +15,34 @@
 # You should have received a copy of the GNU General Public License along
 # with this program; if not, see <http://www.gnu.org/licenses/>.
 
-set -e
+set -ex
```

Comment:
I recommend to not use tracing by default in this script. The `-x` can still be added where needed, e.g. within the calling location

Reply:
removed it